app/controllers/application_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery prepend: true
 
-  include Pundit
+  include Pundit::Authorization
```

Comment:
This was removed some PRs ago and now was added again?

Reply:
We had to do that because we downgraded the Pundit version, so we are upgrading again. And now `include Pundit` was deprecated in favor of `include Pundit::Authorization` on version 2.2.0. PR: https://github.com/varvet/pundit/pull/621